src/units/abilities.cpp

```diff
@@ -941,7 +949,7 @@ std::vector<std::pair<t_string, t_string>> attack_type::abilities_special_toolti
 		}
 	}
 	for(const unit& u : get_unit_map()) {
-		if(!u.has_ability_distant() || u.incapacitated() || &u == self_.get()) {
+		if(!u.has_ability_distant() || u.incapacitated() || &u == self_.get() || u.id() == self_->id()) {
```

Comment:
Should this also use `same_unit`?

Reply:
opps, thanks for have said that to me, i correct that.